fxa-auth-db-server/docs/API.md

```diff
@@ -680,6 +683,56 @@ Content-Type: application/json
 * Status Code : 200 OK
     * Content-Type : 'application/json'
     * Body : `[ ... <see example> ...]`
+* Status Code : 409 Conflict
```

Comment:
Huh, so I see in https://github.com/mozilla/fxa-auth-server/pull/1112/ that we're already using `409` here for other requests, and translating it into 400 errors in the auth-server.  So this is all good, carry on :-)